### ISSUE TITLE
Portability

### ISSUE THREAD

**Issue by tiborepcek** [NoteFly/NoteFly#2]
Can i somehow make NoteFly portable, so that all files written to %user%\AppData\Roaming\ will be written to folder, where NoteFly.exe is located?

**Comment by D9ping** [comment]
Currently there is no command line parameter to override the folder where all files are stored.

**Comment by tiborepcek** [comment]
And do you plan to add that cmd line parameter?

**Comment by D9ping** [comment]
Yes that would be a nice addition have and is not hard to implement. But I cannot promise you when it will be added.
